src/services/userToRoomJoiner.js

```diff
@@ -0,0 +1,61 @@
+const { User, Room } = require('../entities')
```

Comment:
How does 'requiring a folder works'?
I mean how does it gets destructured into User and Room classes?

Reply:
It file name is not passed, it tries to import `dirname/index.js`

Reply:
Oh, so the 'index' can also be used as a 'starting point' not only of the app?
Is it some generally "start by default" name?

Reply:
yes. 
you have a directory with `index.js` file.
being in that directory you can run this file like this
```bash
$ node .
```